src/Passage.sol

```diff
@@ -114,15 +153,27 @@ contract Passage {
         emit Transact(rollupChainId, msg.sender, to, data, value, gas, maxFeePerGas);
     }
 
+    /// @notice Alow/Disallow a given ERC20 token to enter the rollup.
+    function configureEnter(address token, bool _canEnter) external {
+        if (msg.sender != tokenAdmin) revert OnlyTokenAdmin();
```

Comment:
followup work:

bump solidity to `0.8.26` so we can use `require(_, Error)` instead of `if-revert`

Reply:
https://github.com/ethereum/solidity/releases/tag/v0.8.26